apps/meteor/tests/e2e/omnichannel/omnichannel-units.spec.ts

```diff
@@ -84,7 +84,7 @@ test.describe('OC - Manage Units', () => {
 			await expect(poOmnichannelUnits.confirmDeleteModal).toBeVisible();
 			await poOmnichannelUnits.btnConfirmDeleteModal.click();
 			await expect(poOmnichannelUnits.confirmDeleteModal).not.toBeVisible();
-			await expect(page.locator('h3 >> text="No units yet"')).toBeVisible();
```

Comment:
I don't get why this changed

<img width="320" alt="Image" src="https://github.com/user-attachments/assets/77b220f3-6a34-4dfb-b95f-de4742c6fd50" />

Reply:
Reverted